script.js

```diff
@@ -0,0 +1,70 @@
+console.log('js file linked!');
+const resetBut = document.getElementById('resetBut');
+const squares = Array.from(document.getElementsByClassName('square'));
+const playerText = document.getElementById('playerText');
+const spaces= [null, null, null, null, null, null, null, null, null];
+
+console.log(squares);
+
+const oTurn = "O";
+const xTurn = "X";
+
+let currentTurn;
+
+
+const clickCon = (e) => {
+    const id = e.target.id;
+    console.log(id);
+
+    if(!spaces[id]) {
+        spaces[id] = currentTurn;
+        e.target.innerText = currentTurn;
+    
+        currentTurn = currentTurn === oTurn ? xTurn : oTurn;
+    }
+}
+
+squares.forEach((square, index) => {
+    square.addEventListener('click', clickCon);
+    })
+
+    const theWinner = () => {
+        if(boxes[0] === currentTurn) {
+            if(boxes[1] === currentTurn && boxes[2] === currentTurn) {
```

Comment:
I see what you're doing by identifying all 3 winning combos that start with box 1. But I think that it's going to ultimately be a hard way to organize your win cons.